### PR TITLE
Feature/xupvvh 100g

### PR DESCRIPTION
**Pull request by mhrtmnn**
This PR adds 100G support to the Network (sfpplus) plugin on the XUPVVH platform

### CODE REVIEW

File: toolflow/vivado/platform/xupvvh/plugins/sfpplus_100g.tcl [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      connect_bd_net [get_bd_pins $core/sys_reset] [get_bd_pins dclk_reset/peripheral_reset]
+      make_bd_intf_pins_external [get_bd_intf_pins $core/gt_rx]
+      make_bd_intf_pins_external [get_bd_intf_pins $core/gt_tx]
+      connect_bd_net [get_bd_pins $core/drp_clk] [get_bd_pins dclk_wiz/clk_out1]
+      connect_bd_net [get_bd_pins $core/init_clk] [get_bd_pins dclk_wiz/clk_out1]
+
+      # Connect core
+      connect_bd_intf_net [get_bd_intf_pins $core/axis_rx] [get_bd_intf_pins AXIS_RX_${name}]
+      connect_bd_intf_net [get_bd_intf_pins $core/axis_tx] [get_bd_intf_pins AXIS_TX_${name}]
+      #connect_bd_net [get_bd_pins $core/s_axi_aclk] [get_bd_pins dclk_wiz/clk_out1]
```
Comment (wirthjohannes) [inline toolflow/vivado/platform/xupvvh/plugins/sfpplus_100g.tcl:140]: These lines are not necessary?
So they can be removed?

File: toolflow/vivado/platform/xupvvh/xupvvh.tcl [hunk 2]
```diff
@@ -229,7 +229,7 @@ namespace eval platform {
     insert_regslice "dma_host" true "/memory/M_HOST" "/host/S_HOST" "/clocks_and_resets/host_clk" "/clocks_and_resets/host_interconnect_aresetn" ""
     insert_regslice "host_arch" true "/host/M_ARCH" "/arch/S_ARCH" "/clocks_and_resets/design_clk" "/clocks_and_resets/design_interconnect_aresetn" ""
 
-    if {[tapasco::is_feature_enabled "SFPPLUS"]} {
+    if {[get_bd_intf_pins "/network/S_NETWORK"] ne ""} {
```
Comment (wirthjohannes) [inline toolflow/vivado/platform/xupvvh/xupvvh.tcl:232]: What is the reason behind this change?

**Review by wirthjohannes — CHANGES_REQUESTED** [review]
Looks mostly good to me, just the two small comments.
Is there a specific reason why you didn't include the AU280 support?

File: toolflow/vivado/platform/xupvvh/xupvvh.tcl (hunk 2 above)
Comment (mhrtmnn) [reply toolflow/vivado/platform/xupvvh/xupvvh.tcl:232]: Not sure, this is part of your commit (4b617b5337d2870c536dba8f6c967811209dd567). Should we undo it?

Comment (wirthjohannes) [reply toolflow/vivado/platform/xupvvh/xupvvh.tcl:232]: Ah, you're right.
Now I also remember the reason: In the 100G mode the interface `/network/S_NETWORK` does not exist and thus the old version would fail. Is this still the case or does the 100G mode create this interface now?

**Comment by mhrtmnn** [comment]
> Looks mostly good to me, just the two small comments.
> Is there a specific reason why you didn't include the AU280 support?

OK, the idea was not to mix different platforms and protocols into a large PR.
Aurora support and the port to AU280 will follow in separate PRs (that depend on this one)

File: toolflow/vivado/platform/xupvvh/plugins/sfpplus_100g.tcl (hunk 1 above)
Comment (mhrtmnn) [reply toolflow/vivado/platform/xupvvh/plugins/sfpplus_100g.tcl:140]: will fix

File: toolflow/vivado/platform/xupvvh/xupvvh.tcl (hunk 2 above)
Comment (mhrtmnn) [reply toolflow/vivado/platform/xupvvh/xupvvh.tcl:232]: `/network/S_NETWORK` does not exist for 100G, as we do not use the AXI Lite control interface for CMAC (we use ports `ctl_rx_enable` etc instead, which would be removed if we enable the AXI interface)

**Review by wirthjohannes — APPROVED** [review]
LGTM
